EIPS/eip-5XXX.md

```diff
@@ -0,0 +1,341 @@
+---
+eip: 5XXX
+title: Principal Token Standard
+description: A standard for principal tokens (zero-coupon tokens) that are redeemable for a single underlying ERC-20 token at a future timestamp.
+author: Julian Traversa (@JTraversa), Robert Robbins (@robrobbins)
+discussions-to: https://ethereum-magicians.org/t/eip-xxxx-principal-token-standard/1
```

Comment:
Link doesn't work.

Reply:
We haven't created it yet, give us some time. Can I ping you when it's ready for review instead?

Reply:
Fixed in [PR5095](https://github.com/ethereum/EIPs/pull/5095)